topics/single-cell/tutorials/scrna-case_monocle3-rstudio/workflows/main_workflow.ga

```diff
@@ -0,0 +1 @@
olshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1", "tool_shed_repository": {"changeset_revision": "ee98d611afc6", "name": "anndata_inspect", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"input\": {\"__class__\": \"ConnectedValue\"}, \"inspect\": {\"info\": \"obs\", \"__current_case__\": 3}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "0.7.5+galaxy1", "type": "tool", "uuid": "72bdeba6-c986-4fb8-88cb-f4f45cacefeb", "workflow_outputs": [{"label": "Cell metadata (obs)", "output_name": "obs", "uuid": "1a272cd8-5188-4f87-b386-bb6970d431b1"}]}, "3": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1", "errors": null, "id": 3, "input_connections": {"input": {"id": 0, "output_name": "output"}}, "inputs": [], "label": "Inspect AnnData: genes", "name": "Inspect AnnData", "outputs": [{"name": "var", "type": "tabular"}], "position": {"left": 292, "top": 189}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1", "tool_shed_repository": {"changeset_revision": "ee98d611afc6", "name": "anndata_inspect", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"input\": {\"__class__\": \"ConnectedValue\"}, \"inspect\": {\"info\": \"var\", \"__current_case__\": 4}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "0.7.5+galaxy1", "type": "tool", "uuid": "e83f5c1d-8a3b-42e6-8e89-eb21c529373d", "workflow_outputs": [{"label": "Gene metadata (var)", "output_name": "var", "uuid": "23574512-fb66-40c5-ab55-9631cff6a8e2"}]}, "4": {"annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1", "errors": null, "id": 4, "input_connections": {"input": {"id": 0, "output_name": "output"}}, "inputs": [], "label": "Inspect AnnData: expression matrix", "name": "Inspect AnnData", "outputs": [{"name": "X", "type": "tabular"}], "position": {"left": 293, "top": 348}, "post_job_actions": {"RenameDatasetActionX": {"action_arguments": {"newname": "Expression matrix"}, "action_type": "RenameDatasetAction", "output_name": "X"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/anndata_inspect/anndata_inspect/0.7.5+galaxy1", "tool_shed_repository": {"changeset_revision": "ee98d611afc6", "name": "anndata_inspect", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"input\": {\"__class__\": \"ConnectedValue\"}, \"inspect\": {\"info\": \"X\", \"__current_case__\": 1}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "0.7.5+galaxy1", "type": "tool", "uuid": "b6b532e1-f556-4466-8244-6b88fd505cf3", "workflow_outputs": [{"label": "Expression matrix", "output_name": "X", "uuid": "9d95e2ad-9781-4b93-8d40-03dd3d94a4ac"}]}}, "tags": ["name:trajectory_analysis", "name:single_cell", "name:R", "name:RStudio"], "uuid": "bef4e04e-06a5-445c-8379-2869078ff4af", "version": 3}
```

Comment:
looks like just some workflow issues, but otherwise good to go! If you can fix these we can get it merged :)